src/workloads/basic/ReadOnlyMultiThreaded.yml

```diff
@@ -0,0 +1,91 @@
+SchemaVersion: 2018-07-01
+Owner: Product Performance
```

Comment:
Is product perf  the long term owner of this workload?

Reply:
I believe so, since the long term objective is to use this as a stable benchmark and potentially add it to auto-reverter.

Reply:
Sounds good, thanks!